source/gui/nvdaControls.py

```diff
@@ -2,6 +2,7 @@
 # Copyright (C) 2016-2024 NV Access Limited, Derek Riemer, Cyrille Bougot, Luke Davis, Leonard de Ruijter
```

Comment:
could you explain the chmod for `source/gui/logViewer.py`?

Reply:
It was executable. Now it's not.